scan/src/components/validator/DelegatorsTable.re

```diff
@@ -188,3 +260,8 @@ let make = (~address) => {
      }}
   </div>;
 };
+
+[@react.component]
+let make = (~address) => {
+  Media.isMobile() ? renderMobile(~address) : renderDesktop(~address);
```

Comment:
Change `Media.isMobile()` to `let isMobile = Media.isMobile()` then replace to 3 places you wrote.